src/status_im/ui/screens/wallet/choose_recipient/views.cljs

```diff
@@ -61,7 +62,7 @@
                        ;:torchMode     (camera/set-torch camera-flashlight)
                        :onBarCodeRead #(when-not @read-once?
                                          (reset! read-once? true)
-                                         (re-frame/dispatch [:wallet/fill-request-from-url (camera/get-qr-code-data %) :qr]))}]]
+                                         (eip681/parse-uri-handler (camera/get-qr-code-data %) :qr))}]]
```

Comment:
you should dispatch `::wallet/qr-code-request-read` here

Reply:
qr-code-request-read doesn't appear to be a function.  Did you mean `resolve-ens-addresses`?

Reply:
I was talking about dispatching a new event here called like that. And yes that would be an event that calls resolve-ens-address

Reply:
Why not just dispatch `resolve-ens-addresses` directly?  I've done that approach in my next commit and it seems to work fine.